### ISSUE TITLE
[Hunter][Talent] Hunter vs. Wild

### ISSUE THREAD

**Issue by Viryll** [Stormforge-gg/Bugtracker-Fordring#1454]
[wowwiki]https://wowwiki-archive.fandom.com/wiki/Hunter_vs._Wild
[wotlkdb]https://wotlkdb.com/?spell=56341

**Description**: 
**Still doesn't work.**
![image](https://user-images.githubusercontent.com/98886485/174833545-b7976617-be4e-4d4c-89c5-5c3ae851a481.png)
Attack power without 132 stamina buff on owner.
![image](https://user-images.githubusercontent.com/98886485/174833608-768c8896-56a3-4f38-85ed-3bdc342a0f4c.png)
Attack power with 132 stamina buff on owner.
Difference is 11 attack power.
How it should work 132x30/100 ~39,6 + (132x1.1)x30/100*0,24 = 39,6 + 10,45
In short the talent still doesn't work (the pet contribution part).

**Current behaviour**:
Gives attack power to hunter.
**Expected behaviour**: 
Doesn't give attack power to pet. (10/20/30% of hunter stamina).
**Revision**: 
**Revision: 1541**

**Comment by Wolffenstein** [comment]
Fixed in rev 1552.
